### ISSUE TITLE
Unexpected token - build issue with v4.0.3

### ISSUE THREAD

**Issue by elliottbf** [JohannesKlauss/react-hotkeys-hook#837]
Hi there!

I just tried to upgrade from v3.4.7 to v4.0.3 and got a build error that I'm not sure what to do with.

For context I'm using Vite v3.2.3 with typescript and I'm on React 18.

Here's the output from that error:

```
error during build:
10:54:38 AM: SyntaxError: Unexpected token (1:18127) in /opt/build/repo/node_modules/react-hotkeys-hook/dist/index.js
10:54:38 AM:     at Parser.pp$4.raise (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:19420:13)
10:54:38 AM:     at Parser.pp$9.unexpected (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:16714:8)
10:54:38 AM:     at Parser.pp$5.parseExprAtom (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18795:10)
10:54:38 AM:     at Parser.pp$5.parseExprSubscripts (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18587:19)
10:54:38 AM:     at Parser.pp$5.parseMaybeUnary (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18553:17)
10:54:38 AM:     at Parser.pp$5.parseExprOps (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18480:19)
10:54:38 AM:     at Parser.pp$5.parseMaybeConditional (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18463:19)
10:54:38 AM:     at Parser.pp$5.parseMaybeAssign (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18430:19)
10:54:38 AM:     at Parser.pp$5.parseExpression (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:18393:19)
10:54:38 AM:     at Parser.pp$8.parseReturnStatement (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:17034:31)
10:54:38 AM:     at Parser.pp$8.parseStatement (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:16862:37)
10:54:38 AM:     at Parser.pp$8.parseBlock (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:17190:21)
10:54:38 AM:     at Parser.pp$5.parseFunctionBody (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:19246:22)
10:54:38 AM:     at Parser.pp$8.parseFunction (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:17312:8)
10:54:38 AM:     at Parser.pp$8.parseFunctionStatement (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:17012:15)
10:54:38 AM:     at Parser.pp$8.parseStatement (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:16857:17)
10:54:38 AM:     at Parser.pp$8.parseTopLevel (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:16771:21)
10:54:38 AM:     at Parser.parse (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:16543:15)
10:54:38 AM:     at Function.parse (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:16593:35)
10:54:38 AM:     at Graph.contextParse (file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:22959:38)
10:54:38 AM:     at tryParse (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-51c4f80a.js:7213:12)
10:54:38 AM:     at analyzeTopLevelStatements (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-51c4f80a.js:7232:15)
10:54:38 AM:     at Object.transformAndCheckExports (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-51c4f80a.js:9222:68)
10:54:38 AM:     at Object.transform (file:///opt/build/repo/node_modules/vite/dist/node/chunks/dep-51c4f80a.js:9413:41)
10:54:38 AM:     at file:///opt/build/repo/node_modules/rollup/dist/es/shared/rollup.js:22748:40
```

Any ideas about what might be going on here? For now, I've downgraded back to v3.4.7 and the build error is gone.

**Comment by rwollgar** [comment]
Had the same problem. Going back to 4.0.2 and the error disappeared. (react 18.2, webpack 5.75)

**Comment by TDanks2000** [comment]
i still have this issue even on 4.0.2

**Comment by nicksrandall** [comment]
I have a PR in (#839) to fix this. I personally needed the fix ASAP so I published a fork on NPM with the changes: `@nicksrandall/react-hotkeys-hook`

**Comment by JohannesKlauss** [comment]
This should be fixed with version 4.0.4
